react/props_and_component_structure/README.md

```diff
@@ -19,31 +18,29 @@
 * Pass functions through props to manipulate state
 * Articulate the distinction between container and display components
 
-# 1. Introduction to props
+# Introduction to props
 
-In previous lessons, we've seen how to build React components and display them on a website.  However, we had no way to make two different versions of the same component.  For example, we should be able to create a `UserProfile` component with the name "Robert Kahn", then create a separate `UserProfile` component with the name "Vinton Cerf".
+In previous lessons, we've seen how to build React components and display them on a website. However, we had no way to make two different versions of the same component. For example, we should be able to create a `UserProfile` component with the name "Robert Kahn", then create a separate `UserProfile` component with the name "Vinton Cerf".
 
 Another example of this is Amazon's product search. Take a look at, say, a search for t-shirts:
 
 ![amazon](./assets/amazon_screen.png)
 
-Each of these t-shirts represents a separate instance of the **exact same component**. This component is reused in many different places in the application.
+Each of these t-shirt listings represents a separate instance of the **exact same component**. This component is reused in many different places in the application.
 
 In order to have the same component display different information, we can use **props**.
 
 ## Props
 
-Just like React Components have a `state` property, they also have a `props` property.  Props allow components to render other components and to **pass information to those components**.  Components can use props to accurately display information as the state of its parent changes.
+Props allow components to **pass information** to child components that they're rendering. Components can use props to consume data from their parents and render it to the user. This permits us to reuse the same component with relative ease. For example,we don't need distinct `UserProfile` components if each user's profile consumes the same shape of information and renders that information in the same way. We can use the same component and pass down unique user information as props.
 
-Props also allow us to intelligently divide responsibilities between components. We can create components whose only job is to make AJAX requests and store state, and we can have components whose only job is to receive props and render information.
+Props also allow us to intelligently divide responsibilities between components. We can create components whose only job is to fetch, store, and update information, and we can have components whose only job is to receive props and render information. More on that separation of concerns in a future lesson.
```

Comment:
I think it's too soon to be talking about this. If it's all saved for a future lesson, let's delete this paragraph here and focus on the main task of teaching what props are.

Reply:
I'm fine with making this change if necessary, but I think that this concept is one of the most important ones to get across in this unit, and achieving this pattern is one of the main reasons why we'd use props. What I'm worried about is students thinking that props are only for accomplishing a vague idea of "organization" between files without having an understanding of how we should organize our components and data flow.